lib/xml_builder.ex

```diff
@@ -219,7 +219,11 @@ defmodule XmlBuilder do
 
   defp format(:xml_decl, 0, options) do
     encoding = Keyword.get(options, :encoding, "UTF-8")
-    ~s|<?xml version="1.0" encoding="#{encoding}"?>|
+    standalone = case Keyword.get(options, :standalone, "") do
+      true -> ~s| standalone="yes"|
+      other -> other
```

Comment:
I think it should be `other -> ""`, since only `true` is a valid option

Reply:
Yes, makes sense.